src/web/shared/KMZExport.ts

```diff
@@ -10,16 +10,24 @@ import { DriftPacket } from "./shared/JAIAProtobuf";
  * @param {LogTaskPacket} taskPacket The task packet to process into KML features
  * @returns {Promise<string[]>} A promise for the array of strings for each KML feature in `taskPacket`
  */
-async function taskPacketToKMLPlacemarks(taskPacket: LogTaskPacket) {
+async function taskPacketToKMLPlacemarks(taskPacket: TaskPacket | LogTaskPacket): Promise<string[]> {
     var placemarks: string[] = []
 
-    if (taskPacket._scheme_ != 1) {
+    if ('_scheme_' in taskPacket && taskPacket._scheme_ != 1) {
         return []
     }
 
     const formatter = new Intl.DateTimeFormat('en-US', { dateStyle: "medium", timeStyle: "medium" })
-    const date = new Date(taskPacket._utime_ / 1e3)
-    const dateString = formatter.format(date)
+
+    var startTimeString: string
```

Comment:
We could make this line let startTimeString = 'Unknown' and remove the else statement and the const keyword from inside the if statement. Also for the if statement, I think we should use !== undefined for maximum clarity